cypress/e2e/po/pages/home.po.ts

```diff
@@ -18,10 +20,94 @@ export default class HomePagePo extends PagePo {
   }
 
   title(): Cypress.Chainable<string> {
-    return this.self().getId('banner-title').invoke('text');
+    return cy.getId('banner-title').invoke('text');
   }
 
-  changelog(): Cypress.Chainable<string> {
-    return this.self().getId('changelog-banner').invoke('text');
+  graphicBanner(): Cypress.Chainable {
+    return cy.getId('home-banner-graphic');
+  }
+
+  graphicBannerCloseButton(): Cypress.Chainable {
+    return cy.getId('graphic-banner-close').click({ force: true });
+  }
+
+  setLoginPageBanner(): Cypress.Chainable {
+    return cy.getId('set-login-page-banner');
+  }
+
+  closeButton(): Cypress.Chainable {
+    return cy.getId('banner-close').click({ force: true });
+  }
+
+  banner(): Cypress.Chainable {
+    return cy.getId('banner-content');
+  }
+
+  prefPageLink(): Cypress.Chainable {
+    return cy.getId('pref-banner-link');
+  }
+
+  changelog(): Cypress.Chainable {
+    return cy.getId('changelog-banner');
+  }
+
+  whatsNewBannerLink(): Cypress.Chainable {
+    return cy.get('[data-testid="changelog-banner"] a');
```

Comment:
would it be possible to tie this in to the `banner.po` instance above (not in the po, but using that element to find this one)